portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,15 @@ async function showHello() {
     const helloContainer = document.getElementById('hello-container');
     helloContainer.innerText = msg;
 }
+
+async function showContacts() {
+    const responseFromServer = await fetch('/form-handler');
+    const textFromResponse = await responseFromServer.json();
+
+    String = Jsoup.clean(request.getParameter("title"), Whitelist.none());
+    //Pick a random message
```

Comment:
This section needs to be updated.

Reply:
Updated this section in the latest commit.